api/lib/messageCreators/polkadot-transactions.js

```diff
@@ -171,7 +171,7 @@ async function ClaimRewardsTx(senderAddress, api, { rewards }) {
       .forEach((reward) => {
         allClaimingTxs.push(
           api.tx.staking.payoutStakers(
-            reward.validator.operatorAddress,
+            reward.validator,
```

Comment:
this looks like the whole validator Object, not only the address

Reply:
the reward will only hold the address for the validator